### PR TITLE
events are displaying on home view controller and display view controller

### PR DESCRIPTION
**Pull request by darthbatman**

### CODE REVIEW

File: HackIllinois/Controllers/FlowControllers/HILoginFlowController.swift [hunk 1]
```diff
@@ -114,47 +114,97 @@ extension HILoginFlowController {
 
 // MARK: - Login Flow
 extension HILoginFlowController {
-    func populateUserData(loginMethod: HILoginMethod, token: String, sender: HIBaseViewController) {
```
Comment (rauhul) [inline HackIllinois/Controllers/FlowControllers/HILoginFlowController.swift]: This shouldn’t be in the pr

File: HackIllinois/DataSources/HIEventDataSource.swift [hunk 2]
```diff
@@ -27,46 +27,39 @@ final class HIEventDataSource {
...
-                        }
-                        .authorize(with: HIApplicationStateController.shared.user)
-                        .launch()
-                    } else {
-                        completion?()
-                        isRefreshing = false
+        HIEventService.getAllEventsRaw(completed: {events in
+            var actualEvents = [HIAPIEvent]()
+            for event in events {
+                let ev = HIAPIEvent(favorite: event.favorite,
```
Comment (rauhul) [inline HackIllinois/DataSources/HIEventDataSource.swift]: Just update HIAPIEvent, there’s no need to have types of api events

File: HackIllinois/APIService/HIEventService.swift [hunk 3]
```diff
@@ -20,14 +20,28 @@ final class HIEventService: HIBaseService {
...
     }
 
-    static func getAllEvents(active: Bool = false) -> APIRequest<HIAPIEvent.Contained> {
+    static func getAllEvents(active: Bool = false) -> APIRequest<HIAPIEventDated.Contained> {
         let paramaters = ["active": "\(active)"]
-        return APIRequest<HIAPIEvent.Contained>(service: self, endpoint: "", params: paramaters, method: .GET)
+        return APIRequest<HIAPIEventDated.Contained>(service: self, endpoint: "", params: paramaters, method: .GET)
+    }
+
+    static func getAllEventsRaw(completed: @escaping ((_ events: [HIAPIEventDated]) -> Void)) {
```
Comment (rauhul) [inline HackIllinois/APIService/HIEventService.swift]: Use APIManager

File: HackIllinois/DataSources/HIEventDataSource.swift [hunk 4]
```diff
@@ -27,46 +27,39 @@ final class HIEventDataSource {
...
-                            }
-                            completion?()
-                            isRefreshing = false
-                        }
-                        .authorize(with: HIApplicationStateController.shared.user)
-                        .launch()
-                    } else {
-                        completion?()
-                        isRefreshing = false
+        HIEventService.getAllEventsRaw(completed: {events in
```
Comment (rauhul) [inline HackIllinois/DataSources/HIEventDataSource.swift]: Do you not need to fetch locations or favorites anymore

File: HackIllinois/Misc/HIConfiguration.swift [hunk 5]
```diff
@@ -14,15 +14,15 @@ import Foundation
...
+    static let HACKING_END_TIME = Date(timeIntervalSince1970: 1551024000) // Sunday, February 24, 2019 10:00:00 AM GMT-06:00
+    static let EVENT_END_TIME = Date(timeIntervalSince1970: 1551031200) // Sunday, February 24, 2019 12:00:00 PM GMT-06:00
 
-    static let FRIDAY_START_TIME = Date(timeIntervalSince1970: 1519365600) // 2018-02-23T00:00:00-0600 = 2018-02-23T06:00:00+0000
-    static let FRIDAY_END_TIME = Date(timeIntervalSince1970: 1519451999) // 2018-02-23T23:59:59-0600 = 2018-02-24T05:59:59+0000
-    static let SATURDAY_START_TIME = Date(timeIntervalSince1970: 1519452000) // 2018-02-24T00:00:00-0600 = 2018-02-24T06:00:00+0000
-    static let SATURDAY_END_TIME = Date(timeIntervalSince1970: 1519538399) // 2018-02-24T23:59:59-0600 = 2018-02-25T05:59:59+0000
-    static let SUNDAY_START_TIME = Date(timeIntervalSince1970: 1519538400) // 2018-02-25T00:00:00-0600 = 2018-02-25T06:00:00+0000
-    static let SUNDAY_END_TIME = Date(timeIntervalSince1970: 1519624799) // 2018-02-25T23:59:59-0600 = 2018-02-26T05:59:59+0000
+    static let FRIDAY_START_TIME = Date(timeIntervalSince1970: 1550815200) // 2018-02-23T00:00:00-0600 = 2018-02-23T06:00:00+0000
```
Comment (rauhul) [inline HackIllinois/Misc/HIConfiguration.swift]: Updates these comments like the ones you have above

File: HackIllinois/Models/CoreDataModels/Event+CoreDataClass.swift [hunk 6]
```diff
@@ -18,12 +18,11 @@ public class Event: NSManagedObject {
     convenience init(context moc: NSManagedObjectContext, event: HIAPIEvent, locations: NSSet) {
         guard let entity = NSEntityDescription.entity(forEntityName: "Event", in: moc) else { fatalError() }
         self.init(entity: entity, insertInto: moc)
-        favorite = event.favorite
-        end = event.end
-        id = event.id
-        info = event.info
-        name = event.name
-        start = event.start
+        self.favorite = event.favorite
```
Comment (rauhul) [inline HackIllinois/Models/CoreDataModels/Event+CoreDataClass.swift]: Considered bad style to add self when its unnecessary, unless a majority of properties are shadowed (which is not the case here)

File: HackIllinois/DataSources/HIEventDataSource.swift (hunk 4 above)
Comment (darthbatman) [reply HackIllinois/DataSources/HIEventDataSource.swift]: dont need to fetch locations anymore, working on favorites

**Comment by rauhul** [comment]
See #162 which is a properly rebased version of this pr

**Pull request closed without merging**